creator-node/src/utils.js

```diff
@@ -645,6 +645,25 @@ async function runShellCommand(command, args, logger) {
   })
 }
 
+/**
+ * Calls `fn` with retries; throws error if failed after all, else returns `fn`'s return value
+ */
+async function asyncRetry(fn, { retries }) {
```

Comment:
rolled this myself instead of using `async-retry` bc for some reason tests kept failing with `async-retry`, some issue with proxyquire + async-retry

Reply:
yeah its implemented repeatedly and differently all over CN, wanted to at least standardize it here for consumption moving fwd - ideally we shouldn't need this at all and could just use `async-retry` but was having some issue 🤷‍♂️

Reply:
what was the issue? I can take a look. I rather spend a little more time trying to figure it out than adding code that performs similarly

Reply:
yeah would be super down to switch if you can figure it out - i spent a decent bit of time and couldn't get it to work so moved on

see error screenshot:
<img width="864" alt="Screen Shot 2022-01-26 at 2 15 45 PM" src="https://user-images.githubusercontent.com/3323835/151230991-72ffdbf2-6431-4326-b6ee-c1e8f20b09f7.png">

vs success with my function
<img width="861" alt="Screen Shot 2022-01-26 at 2 16 46 PM" src="https://user-images.githubusercontent.com/3323835/151231141-c9e487a3-bfdc-4c51-8de5-b2183661ed30.png">

the test that fails is `Throws error primaryFilesHashForRange can't be retrieved` in `src/snapbackSM/snapbackSM.test.js`

Reply:
crisis averted - vicky got `async-retry` to work, swapped over to that